staff/johana-ludwig/cv/index.html

```diff
@@ -19,7 +21,7 @@ <h2>Online Marketing y Frontend Development</h2>
             <tr>
                 <td></td>
                 <td>
-                    <h3>Hola, soy Johana Ludwig</h3>
+                    <header><h3>Hola, soy Johana Ludwig</h3></header>
```

Comment:
Sorry Johana! 😞  my mistake. I asked to Manu and you can put the h3 tag without `header` tag. In fact, the header tag is when you have more elements besides the h3

Reply:
cool thanks! i will change it again ;)